root/static/scripts/common/MB/release.js

```diff
@@ -61,7 +61,7 @@ $(function () {
       .addClass('loading-message')
       .text(l('Loading...'));
 
-    var mediumId = this.getAttribute('data-medium-id');
+    var mediumId = element.data('medium-id');
 
     $.get('/medium/' + mediumId + '/fragments')
```

Comment:
The only concern I have here is that if you have a release with say 100 discs, there isn't any sort of rate limiting here that I can see. So we should probably update this to use root/static/scripts/common/utility/request.js too. (Sorry for the quick approval revert, the rest looks great at least. :D)

Reply:
Ok, changed it to use request.